src/admin/quick-lane/views/QuickLaneOverview.tsx

```diff
@@ -138,6 +152,13 @@ const QuickLaneOverview: FunctionComponent<QuickLaneOverviewProps> = ({ user })
 		fetchQuickLanes();
 	}, [fetchQuickLanes]);
 
+	useEffect(() => {
+		setQuery({
+			sort_column: query.sort_column || QUICK_LANE_DEFAULTS.sort_column,
+			sort_order: query.sort_order || QUICK_LANE_DEFAULTS.sort_order,
+		});
+	});
```

Comment:
This seems to cause an infinite render loop, did you mean to write this?
```suggestion
	}, [setQuery]);
```